### PR TITLE
adjust code to new ticket shop

### PR DESCRIPTION
**Pull request by MartinLangbecker**
relates to https://github.com/derhuerst/generate-db-shop-urls/issues/47

Test data has not yet been fixed since I'm still missing some information.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -59,16 +59,16 @@ const generateDbShopLink = async (outbound, opt) => {
 	}
 
 	const orig = outbound.legs[0].origin
-	const originId = orig.station && orig.station.id || orig.id || orig
+	const originId = orig.station?.id || orig.id || orig
```
Comment (derhuerst) [inline index.js:62]: This raises the minimum Node.js version to 14.x, but currently it is 10.x. Therefore, this is a breaking change (which is fine).

Comment (MartinLangbecker) [reply index.js:62]: Didn't even think about that, to be honest. Just made it easier for me to read and understand. I can change it back or leave it as is, either way is fine with me.

**Comment by derhuerst** [comment]
I have adapted your commit and pushed it as 1c43d72. Thanks!

**Comment by derhuerst** [comment]
Published as `generate-db-shop-urls@4.0.0`! 🎉

**Pull request closed without merging**